tensorflow/python/training/momentum.py

```diff
@@ -31,13 +31,16 @@ class MomentumOptimizer(optimizer.Optimizer):
   """
 
   def __init__(self, learning_rate, momentum,
-               use_locking=False, name="Momentum", use_nesterov=False):
+               use_locking=False, use_nesterov=False, name="Momentum"):
```

Comment:
Thanks for the doc fix!  Can you restore the ordering of the argument, in case people are not using keyword arguments?